client/src/components/Signup.js

```diff
@@ -858,7 +875,7 @@ export default class Signup extends React.Component {
                         >
                         </Form.Dropdown>
                         <Form.Field
-                            required={true}
+                            //required={true}
```

Comment:
I'm pretty sure required can be uncommented - all it does is add the red asterisk next to the field label